pages/documentation/developer/tutorials/emf.md

```diff
@@ -0,0 +1,344 @@
+```yaml
+balances-adjustment-validation-parameters:
+  data-exchanges-path: path-PEVF #complete with the path to your PEVF file
+
+  igm-paths: 
+    - IGM1-name, path-to-IGM1 #complete with the name of your first IGM and the path to the zipfile
+    - IGM2-name, path-to-IGM2 #complete with the name of your second IGM and the path to the zipfile
+    
+  output-dir: path-to-directory
+```
+
+## Create `BalancesAdjustmentValidationParameters` class to load the parameters from the configuration file
+First, we create a java class called `BalancesAdjustmentValidationParameters` containing the parameters that we want to use, such as the paths of the IGMs, the path to the PEVF file and the output directory. All these parameters are read from the configuration file created before. This class has a method to load the parameters from the configuration file. 
+
+The IGM paths are stored in a HashMap and the output directory and PEVF in Strings. You can also create the getter/setter associated with each variable. Then, you need create a method `load` that will read the inputs from the configuration file and store the data in each variable. If you have difficulties creating this class, you can check the result `powsybl-tutorials/emf` from the Github repository.
+If you want to learn more about the configuration file and how it is  handled by Powsybl, you can find more details [here](../../user/configuration/index.md).
+
+Now with this class, we are able to read the extra parameters from the `config.yml` file. We will move on to create the `EmfTutorial` main class, that will perform the merging and the balance computation.
+
+## Create `EmfTutorial` class to run the computation
+
+In everything that follows, if you have difficulties creating the methods, you can refer to the code in `emf`, in the Github repository.
+
+### Set the load flow parameters
+Once you have created the `EmfTutorial` class, just before the main method, define the variable `LOAD_FLOW_PARAMETERS` of type `LoadFlowParameters`. 
+This variable gathers all the parameters to be used in the load flow pre-processing of the IGMs, in the load flow calculation on the CGM and for the balance computation iterations. In this tutorial, we set the initial voltage value to the DC values, the balance type to proportional to the maximum active power target, the `ReadSlackBus` to true (the slack bus defined in each IGM). The tap changers regulation is also set to true (either for phase tap changers than for ratio tap changers), and the power flow must be computed over all connected components. These parameters are chosen to comply with the European merging function. For more information on the power flow parameters available and how to implement them, you can visit this [page](pages/documentation/simulation/powerflow/openlf.md). In case of non-convergence, these parameters can be relaxed.
```

Comment:
initial voltage value to the DC values -> initial angles values to angles computed through a DC power flow

Reply:
proportional to the maximum active power target -> proportional to the maximum active power target of generators